test/images/php-postgres/apache2.conf

```diff
@@ -0,0 +1,242 @@
+# This is the main Apache server configuration file.  It contains the
```

Comment:
can we nuke this file? for sqlmap tests it works without

Reply:
Yes. I can delete this file.